src/connector.rs

```diff
@@ -31,7 +31,7 @@ impl Connector {
     }
 
     fn id(manifest: &Manifest, config: &Config) -> String {
-        format!("{}-{}-{}-{:?}", manifest.author, manifest.name, manifest.version, config.kafka_env)
+        format!("{}-{}-{}-{}", manifest.author, manifest.name, manifest.version, config.kafka_env)
```

Comment:
why not just use as_str()?

Reply:
Already inplemented Display interface so dont need to convert it to str

Reply:
It looks more standard

Reply:
sounds good